src/main/java/net/glease/ggfab/mte/MTE_AdvAssLine.java

```diff
@@ -654,12 +692,38 @@ public boolean checkRecipe(ItemStack aStack) {
         if (GT_Values.D1) {
             GT_FML_LOGGER.info("Recipe successful");
         }
+        return CheckRecipeResultRegistry.SUCCESSFUL;
+    }
+
+    @Override
+    public boolean supportsVoidProtection() {
         return true;
     }
 
     @Override
-    public GT_Recipe.GT_Recipe_Map getRecipeMap() {
-        return null;
+    public ButtonWidget createVoidExcessButton(IWidgetBuilder<?> builder) {
```

Comment:
Why not make use of `getAllowedVoidingModes` you added?

Reply:
because this commit predates its merge. now it's good time to switch over